src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,13 +1,130 @@
 import NextAuth from 'next-auth';
 import { NextApiHandler } from 'next';
 import { PrismaAdapter } from '@next-auth/prisma-adapter';
-import prisma from 'src/prisma';
+import { PrismaClient } from '@prisma/client';
 import GitHubProvider from 'next-auth/providers/github';
 import GoogleProvider from 'next-auth/providers/google';
+import CredentialsProvider from 'next-auth/providers/credentials';
+import { compare } from 'bcryptjs';
+import type { NextAuthOptions } from 'next-auth'
 
+const prisma = new PrismaClient();
 
+export const authOptions: NextAuthOptions = {
+  debug: true,
+  session: {
+    strategy: 'jwt', // force jwt instead of Prisma adapter default (db sessions)
+    // this also should resolve the final roadblock to actually using credential sessions, but we've been down that road unsuccessfully maybe too many times haha
```

Comment:
Let's clean this file up (comments and duplicate values) now that you've solidified the multiple providers